cif_core.dic

```diff
@@ -9756,6 +9756,44 @@ save_exptl_absorpt.process_details
 
 save_
 
+save_exptl_absorpt.special_details
+
+    _definition.id                '_exptl_absorpt.special_details'
+
+    loop_
+      _alias.definition_id
+         '_exptl_absorpt_special_details'
+         '_exptl.absorpt_special_details'
```

Comment:
The `_exptl.absorpt_special_details` alias should not be included as it was probably not used anywhere before. Aliases of other data items from this category that have the form `_exptl.<something>` (e.g. `_exptl.absorpt_coefficient_mu`) seem to have been taken from the mmCIF dictionary and are thus more meaningful. If needed, we can always add this alias in the future.

Reply:
Agree.